ethportal-api/src/types/cli.rs

```diff
@@ -256,7 +301,7 @@ impl TrinConfig {
 
     pub fn new_from<I, T>(args: I) -> Result<Self, clap::Error>
     where
-        I: Iterator<Item = T>,
+        I: IntoIterator<Item = T>,
```

Comment:
I do really like this change. It also added noise to a big PR. I think it belongs in its own commit/PR.